.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu, windows ]
-        node-version: [14.x, 16.x, 18.x]
```

Comment:
because of @typescript-eslint/scope-manager

Reply:
This'll require a major release to drop node 14, cc @bmish

Reply:
I can also try to downgrade to 5.62.0.
They dropped 14 with the 6.0.0 release

Reply:
i downgraded as i also needed lower version for testing with typed eslint